Content.Server/Access/Components/IdCardConsoleComponent.cs

```diff
@@ -256,6 +256,8 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
         [Verb]
         public sealed class EjectPrivilegedIDVerb : Verb<IdCardConsoleComponent>
         {
+            public override bool AlternativeInteraction => true;
```

Comment:
this isn't a direct review you have to address but just wanna talk about it somewhere for posterity so people understand the impetus/potential future implications here

in BYOND/ss13 'verbs' (context menu items / 'commands') are more first class citizens, and most interactions are defined first in verbs and then interactions simply invoke the verb (afaict)

in SS14 verbs are more of a secondary thing that usually go through component/entitysystem methods; this PR -sort of- changes that by having alternative interactions directly go through verbs when possible. it should be considered in the future whether we want activate-interactions (clicking, E) and use-interactions (Z) to go through this as well, which was discussed here and which PJB agreed might be okay: 

![image](https://user-images.githubusercontent.com/19853115/130172753-f6a0c7bb-006f-4c34-8bc5-697a607c7b97.png)

https://discord.com/channels/310555209753690112/311537926376783886/877458898691457065